### ISSUE TITLE
DisplayHook warning broken

### ISSUE THREAD

**Issue by rgbyrnes** [ipython/ipython#9964]
`IPython.core.displayhook.DisplayHook.__init__` still calls `warn` with a `level` keyword from `IPython.utils.warn`, even though it now imports the standard `warn` from `warnings`, which doesn't support `level`.

Simple reproducer:

```
from IPython.core.displayhook import DisplayHook
DisplayHook(cache_size=2)
```

**Comment by takluyver** [comment]
Thanks, good catch.

**Comment by srinivasreddy** [comment]
raised PR #10023

**Comment by takluyver** [comment]
Fixed by #10023
